test/parallel/test-stream-base-typechecking.js

```diff
@@ -1,7 +1,14 @@
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
+const net = require('net');
 
-assert.throws(() => {
-  process.stdout.write('broken', 'buffer');
-}, /^TypeError: Second argument must be a buffer$/);
+const server = net.createServer().listen(0, common.mustCall(() => {
```

Comment:
Why not add the old one as well under an gourd
`if (process.stdout && process.stdout.write)`
or
```
const {Stream} = require('stream');
if (process.stdout instanceof Stream)
```